### ISSUE TITLE
Domain `jwasm.info`

### ISSUE THREAD

**Issue by XVilka** [Baron-von-Riedesel/JWasm#11]
Since I started the fork some time ago thinking the development was abandoned and obviously it is continued here and active, do you need me to transfer the domain I registered to you? Let me know.

I will archive then all repositories in https://github.com/JWasm

**Comment by Baron-von-Riedesel** [comment]
> do you need me to transfer the domain I registered to you?

No, I'm not that much interested in jwasm anymore - probably I will restrict my activities to bugfixes.
